examples/advanced/cifar10/pt/cifar10-real-world/cifar10_fedavg_he/job.py

```diff
@@ -83,7 +83,7 @@ def main():
         min_clients=n_clients,
         num_rounds=num_rounds,
         initial_model=ModerateCNN(),
-        train_script=os.path.join(os.path.dirname(__file__), "../../../src/client.py"),
+        train_script=os.path.join(os.path.dirname(__file__), "client.py"),
```

Comment:
can we just put "client.py"?

Reply:
Then you would have to cd to the `cifar10_fedavg_he` folder to run `python job.py`. This way, it can be run from anywhere.